src/part05.rs

```diff
@@ -47,7 +47,8 @@ impl BigInt {
         }
     }
 
-    // We can convert any vector of digits into a number, by removing trailing zeros. The `mut`
+    // Any vector of digits, which meets the structure of BigInt's `data` field, can be easily
```

Comment:
Sure, that would have led me into the right direction.

Thank you for still maintaining this tutorial despite your busy work. I've read about your accomplishment during the all hands week. Keep on with your great work!